deployment/terraform/info.go

```diff
@@ -22,11 +22,16 @@ func (t *Terraform) Info() error {
 func (t *Terraform) displayInfo(output *terraformOutput) {
 	fmt.Println("==================================================")
 	fmt.Println("Deployment information:")
-	fmt.Println("Mattermost URL: http://" + output.Proxy.Value.PublicDNS)
-	fmt.Println("App Servers:")
-	for _, instance := range output.Instances.Value {
-		fmt.Println("- " + instance.Tags.Name + ": " + instance.PublicIP)
+	if len(output.Proxy.Value) > 0 {
+		fmt.Println("Mattermost URL: http://" + output.Proxy.Value[0].PublicDNS)
+		fmt.Println("App Servers:")
+		for _, instance := range output.Instances.Value {
+			fmt.Println("- " + instance.Tags.Name + ": " + instance.PublicIP)
+		}
+	} else {
+		fmt.Println("Mattermost URL: http://" + output.Instances.Value[0].PublicDNS + ":8065")
```

Comment:
If there's no proxy we skip showing the app server IP but only print the MM URL.
I think we should still show it as a single app server.